src/transaction.js

```diff
@@ -600,13 +604,23 @@ const transaction = {
    *
    * @return {Object} Object with the parameters {'txMinWeight', 'txWeightCoefficient', 'txMinWeightK'}
    *
+   * @throws {ConstantNotSet} If the weight constants are not set yet
+   *
    * @memberof Transaction
    * @inner
    */
   getTransactionWeightConstants() {
-    return {'txMinWeight': parseFloat(storage.getItem('wallet:txMinWeight')),
```

Comment:
Why don't we use a single key to all of them?
The basic usage would be:
```
const txParams = getTransactionWeightContants();
txParams.txMinWeight
```

Reply:
Done